src/modules/kvs/cache.h

```diff
@@ -89,6 +89,11 @@ int cache_entry_set_errnum_on_notdirty (struct cache_entry *entry, int errnum);
 int cache_entry_wait_notdirty (struct cache_entry *entry, wait_t *wait);
 int cache_entry_wait_valid (struct cache_entry *entry, wait_t *wait);
 
+/* Get the blobref this entry is stored in the cache with.  Returns
+ * blobref on success, NULL if not yet stored in the cache.
```

Comment:
Comment about NULL return is out of date

Reply:
doh!  thanks for the catch

Reply:
hmmm, now that I think of it, with the change to ```cache_entry_create()``` taking a blobref, and ```cache_insert()``` no longer taking a ref, ```cache_insert()``` doesn't have to return an int.  It can go back to returning a void.  That would remove a lot of changes I did.

But then again, things could change in the future.  Perhaps best to have ```cache_insert()``` return an int still.

Reply:
I think leaving the possibility of returning an error is probably not a bad thing, since it's somewhat dependent on the hash implementation whether an error is possible.